### ISSUE TITLE
Failed to load project

### ISSUE THREAD

**Issue by thpoll83** [theacodes/kicanvas#88]
I tried to open  my KiCad project with the following URL: https://github.com/thpoll83/PolyKybd/tree/master/poly_kybd But it does not load anything

**Comment by theacodes** [comment]
It looks like this does eventually load, but there's a *lot* of files here. It throws up warning about net ties, which isn't surprising since I haven't fully implemented KiCAD 8's new features.

<img width="1779" alt="Screenshot 2024-05-21 at 11 15 22 AM" src="https://github.com/theacodes/kicanvas/assets/250995/28cbc996-9764-4d82-9678-09608c6fba43">

**Comment by thpoll83** [comment]
Right, its a bit messy :) And I use hierarchical schematics quite a bit. However, I'm still only using KiCad7, maybe some later version already supports net ties..?

**Comment by theacodes** [comment]
Oh interesting 🤔
